x-pack/plugins/case/common/api/cases/user_actions.ts

```diff
@@ -14,6 +14,7 @@ import { UserRT } from '../user';
 const UserActionFieldRt = rt.array(
   rt.union([
     rt.literal('comment'),
+    rt.literal('connector_id'),
```

Comment:
nit -> I think is better to describe the action with a better word like `change-connector`, `connector-changed` etc.

Reply:
but we are really updating the connector_id field

Reply:
i'd like to leave it. its updating the field `connector_id`, there is no field `connector`

Reply:
Ok! I am not against it. For the sake of argument this a verb that describes an action, not a field. So the user creates a comment, updates the description, **updates the connector** etc.

Reply:
lol, maybe I did not do that correctly but the goal here was to be a field ;)